### DISCUSSION TITLE
Add intermediate DPI scale between 4k and QHD, and/or add global font size adjustment

### DISCUSSION THREAD

**Discussion by SWu** [TT-ReBORN/Georgia-ReBORN#185] [Ideas]
I'm on a 4k monitor, but I currently find that the 4k scaling option is just a bit too large for my taste. I'm using the QHD scale instead, but I feel like some elements like the playback buttons and top bar are just a bit too small, even when adjusting the font/button sizes to the largest options. Is it possible to add an intermediate DPI scale between the two? There seems to be not much difference between HD and QHD, and then a big jump up to 4k. Additionally, would it be possible to add a shortcut to globally increase/decrease all font sizes across the entire theme instead of having to individually go into each submenu and changing them separately? That would also help with tweaking the overall size of things given a starting display scale setting.

**Comment by TT-ReBORN** [comment]
Hi @SWu,

based on the DPI setting, if you set a large Windows display scaling ( 200% or more ) then the player will be automatically enlarged.
So when using 4K mode, you can either reduce the Windows display scaling or use QHD mode and set a higher display scaling.
Keep in mind, when changing the Windows display scaling settings, foobar must be restarted to take effect.

Maybe I will implement an extra manual scaling -50%, -40%, -30%, -20%, 10%, 0%, 10%, 20%, 30%, 40%, 50% in future versions.
This is not a quick job and the implementation is quite a lot of work, due to handling correct adjustments of player sizes, 
various UI elements and font sizes.

-TT

**Comment by TT-ReBORN** [comment]
This has been implemented and is now available in the master:
https://github.com/TT-ReBORN/Georgia-ReBORN/archive/refs/heads/master.zip

 ### Display Scaling
 - Implemented display scaling
   * Can be accessed in top Menu > Options > Display > Scaling.
   * You can also use the keyboard shortcuts "Ctrl and +" for increase or "Ctrl and -" for decrease and "Alt and +/-" for reset.
   * It will scale the entire user interface from 50% - 150% for each individual player size and active display resolution mode.
   * This means when display resolution mode is 4K and player size is large, you can scale it up to 150% and it is ready for 8K resolution!
   * Additionally you can still adjust and fine tune font and button sizes when using the new scaling feature.
   * Scale settings will be saved when making a new backup and will be restored.
